### ISSUE TITLE
update lottie version in expo doctor

### ISSUE THREAD

**Issue by wcastand** [expo/expo-cli#4458]
### Summary

i'm trying to update to sdk 45 for a little while now(since the release) and waited hoping it would be fixed but doesn't seems to be on the menu (i asked on expo discord the process for this but got no answer). so i'm making an issue to see what's the process to update the `lottie-react-native` version to the latest to remove the warning about ViewProps.

i also noticed a warning on android for stripe and the big difference in version between what expo expect and the current npm release on `@stripe/stripe-react-native` (0.6.1 and publish version is 0.14.0). Getting this warning on android when i upgrade to sdk 45 : stripe/stripe-react-native/issues/1032

### Environment

❯ npx expo-env-info
Need to install the following packages:
  expo-env-info
Ok to proceed? (y) y

  expo-env-info 1.0.4 environment info:
    System:
      OS: macOS 12.4
      Shell: 3.4.1 - /opt/homebrew/bin/fish
    Binaries:
      Node: 16.14.2 - ~/.nvm/versions/node/v16.14.2/bin/node
      Yarn: 3.2.0 - ~/.yarn/bin/yarn
      npm: 8.5.0 - ~/.nvm/versions/node/v16.14.2/bin/npm
      Watchman: 2022.05.30.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.11.3 - /opt/homebrew/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    IDEs:
      Android Studio: 2021.1 AI-211.7628.21.2111.8193401
      Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
    npmPackages:
      babel-preset-expo: ~9.1.0 => 9.1.0
      expo: ^45.0.6 => 45.0.6
      react: 17.0.2 => 17.0.2
      react-dom: 17.0.2 => 17.0.2
      react-native: 0.68.2 => 0.68.2
    npmGlobalPackages:
      eas-cli: 0.52.0
      expo-cli: 5.4.12
    Expo Workflow: managed

### Please specify your device/emulator/simulator platform, model and version

mac air m1, emulator is iphone 12 mini and pixel a4 (both emulator)

### Error output

no error just a warning.

### Reproducible demo or steps to reproduce from a blank project

show the warning from lottie and added the one from stripe at the same time.

i'm not sure about stripe but for lottie updating to 5.1.3 is enough to remove the warning and still works for me when i tried.

https://snack.expo.dev/@greengot/lottie-warning-example

**Comment by tomasdanjonsson** [comment]
Been having the same issue with lottie on sdk 45. Here is an open issue about this at [lottie-react-native](https://github.com/lottie-react-native/lottie-react-native/issues/887)

Like @wcastand said version `5.1.3` seems to be working fine, in the meantime for anyone using `5.0.1` and being annoyed by the yellow box you can ignore the warning:

```js
import { LogBox } from 'react-native';

LogBox.ignoreLogs(['ViewPropTypes will be removed']);
```

**Comment by brentvatne** [comment]
sdk 46 updates the lottie version https://blog.expo.dev/expo-sdk-46-beta-is-now-available-9dfee4040aa7
